lua/plugins/which_key.lua

```diff
@@ -1,22 +1,30 @@
 return { -- Useful plugin to show you pending keybinds.
-    'folke/which-key.nvim',
-    event = 'VimEnter', -- Sets the loading event to 'VimEnter'
-    config = function() -- This is the function that runs, AFTER loading
-      require('which-key').setup()
+  'folke/which-key.nvim',
```

Comment:
Odd, We might need a [stylua formatter setup](https://github.com/JohnnyMorganz/StyLua). @sanchayanghosh Can you look into this please?